tests/integration/helpers/diff-to-document-test.ts

```diff
@@ -0,0 +1,20 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'codecrafters-frontend/tests/helpers';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Helper | diff-to-document', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it converts a diff to a document object', async function (assert) {
+    await render(hbs`
+      {{#let (diff-to-document ' 1234\n+2345\n 6789\n-0123\n') as |doc|}}
+        <pre data-test-current-document>{{doc.current}}</pre>
```

Comment:
Like the testing strategy here! Simple end-to-end instead of invoking the helper directly.